rockset/resource_user_test.go

```diff
@@ -77,7 +77,7 @@ func testAccCheckRocksetUserDestroy(s *terraform.State) error {
 		// An error would mean we didn't find the key, we expect an error
 		if err == nil {
 			// We did not get an error, so we failed to delete the key.
-			return fmt.Errorf("User %s still exists.", email)
+			return fmt.Errorf("user %s still exists", email)
```

Comment:
What's with lowercasing and removing punctuation from all of these errors? Is this meant to fit in a longer sentence or something?

Reply:
it is the go-idiomatic error message, and as you often use error wrapping, e.g.
```
if err != nil {
  return fmt.Errorf("failed to write to database: %w", err)
}
```

it read cleaner when you use just lowercase and no period